.circleci/config.yml

```diff
@@ -713,3 +729,13 @@ workflows:
               ignore: /.*/
             tags:
               only: /^v.*/
+
+      # Documentation website jobs.
+      #
+      # Publish the documentation website to GitHub Pages.
+      # Only do it for master as tagged releases are supposed to tag their own version of the
+      # documentation in the release commit on master before they go out.
```

Comment:
The problem is we will always forget to generate the documentation for a release... You should add a check that richie's version number is not bumped before the corresponding version of the documentation exists. 
If checking the presence of a directory, something like:
```bash
BACKEND_VERSION=$(cat setup.cfg | grep "version" | cut -d" " -f3)
[ -d "docs/releases/$(BACKEND_VERSION)" ]
```

Or if checking the presence of the current richie version in docs/website/versions.json, something like:
```bash
BACKEND_VERSION=$(cat setup.cfg | grep "version" | cut -d" " -f3)
cat docs/website/versions.json | grep "$(BACKEND_VERSION)"
 
```

Reply:
Or even better, do it in another PR after this one is merged.

Reply:
Done